TMessagesProj/src/main/res/values/cg_strings.xml

```diff
@@ -270,19 +269,19 @@
     <string name="CG_ToBeginning">To Beginning</string>
     <string name="CG_ToLinkedChat">Linked Chat</string>
     <string name="CG_FakeUserPhone">Fake phone number</string>
-    <string name="CG_FakeUserPhoneSummary">show fake phone number in drawer and settings (hide phone number preference must be disabled)</string>
+    <string name="CG_FakeUserPhoneSummary">Display a fake phone number in drawer and settings. (Note that the Hide phone number preference must be disabled for this to work)</string>
```

Comment:
Is it possible to somehow shorten this text so that it is not longer than the original?

Reply:
Sure